pyproject.toml

```diff
@@ -87,6 +87,8 @@ trace_server = [
   "openinference-semantic-conventions>=0.1.17",
   # For emoji shortcode support in Feedback
   "emoji>=2.12.1",
+  # ClickHouse support
+  "clickhouse_connect==0.7.0",
```

Comment:
this was added because the docs weren't building correctly, due to a very long stack trace that ends in the errors module being loaded, which has clickhouse specific errors